src/dataset-builder/CohortEditor.ts

```diff
@@ -99,7 +99,7 @@ export const CriteriaView = (props: CriteriaViewProps) => {
               deleteCriteria(criteria);
             },
           },
-          [icon('minus-circle-red', { size: 24, style: { color: colors.danger() } })]
+          [icon('trash-circle-filled', { size: 35, style: { color: colors.danger() } })]
```

Comment:
Changing from minus icon to trash icon 
Figma:
![Screenshot 2024-04-01 at 5 28 34 PM](https://github.com/DataBiosphere/terra-ui/assets/63474660/a547181a-334b-4c28-bbc4-4c1ea117e77a)

Also, check out video at the end to see this in PR

Reply:
I thought this was small enough change to keep in this PR